tests/conftest.py

```diff
@@ -252,29 +252,37 @@ def pytest_generate_tests(metafunc):
 
 
 def pytest_collection_modifyitems(config, items):
-    if config.getoption("--aws-integration"):
-        # --aws-integration given in cli: do not skip aws-integration tests
-        return
-    if config.getoption("--docs-tests"):
-        # --docs-tests given in cli: do not skip documentation integration tests
-        return
-    if config.getoption("--cloud"):
-        # --cloud given in cli: do not skip GX Cloud integration tests
-        return
```

Comment:
Previously, if any of these were set, the skip process would end early. This logic has been updated (thank you @billdirks!)